apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -84,6 +84,21 @@ type FlowCollectorSpec struct {
 	// +optional
 	// +k8s:conversion-gen=false
 	Exporters []*FlowCollectorExporter `json:"exporters"`
+
+	// `networkPolicy` define network policy settings for netobserv
+	// +k8s:conversion-gen=false
+	NetworkPolicy NetworkPolycy `json:"networkPolicy,omitempty"`
+}
+
+type NetworkPolycy struct {
+	// Set `deploy` to `false` to disable network policy deployment. It is enabled by default.
+	// +optional
+	Deploy *bool `json:"deploy,omitempty"`
+
+	// `additionalNamespaces` contains the interface names from where flows are collected. If empty, the agent
```

Comment:
hmm I guess it's not the good text  here :-)

Reply:
We should mention here in particular what users need to care about:
- If they use Kafka and installed it in a separate namespace, they must add it here
- Same for any exporter

We can expect users misconfigurations here... Probably something that we'll have to worry about in docs & troubleshooting

Reply:
Noted as something to add to docs :)

Reply:
Cool, thanks Mehul!